client_connection/client_connection.go

```diff
@@ -0,0 +1,55 @@
+package client_connection
+
+import (
+	"errors"
+	"github.com/mysterium/node/service_discovery/dto"
+)
+
+type State string
+
+const (
+	NOT_CONNECTED = State("NOT_CONNECTED")
+	NEGOTIATING   = State("NEGOTIATING")
+	CONNECTED     = State("CONNECTED")
+)
+
+var (
+	ALREADY_CONNECTED = errors.New("already connected")
+)
+
+type Status struct {
+	State     State
+	SessionId string
+}
+
+type Manager interface {
```

Comment:
Looks like filename should be `client_connection/client_connection.go` -> `client_connection/manager.go`

Reply:
Noted and fixed.